README.markdown

```diff
@@ -7231,6 +7233,40 @@ This feature was first introduced in the `v0.5.0rc1` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+tcpsock:receiveany
+------------------
+**syntax:** *data, err = tcpsock:receiveany(max)*
+
+**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;*
+
+Returns any data received by the connected socket, at most `max` bytes.
+
+This method is a synchronous operation just like the [send](#tcpsocksend) method and is 100% nonblocking.
+
+In case of success, it returns the data received; in case of error, it returns `nil` with a string describing the error.
+
+If the received data is more than this size, this method will return with exactly this size of data.
+The remain received data could be returned in the next `receiveany` call.
```

Comment:
Grammar: "The remain received data" => "The remaining data in the underlying receive buffer"

"in the next `receiveany` call" => "in the next reading operation" (it should not necessarily be another `receiveany` call, right? it could also be `receive()` or `receiveuntil()`).